lib/pacemaker/pcmk_sched_colocation.c

```diff
@@ -361,7 +367,7 @@ unpack_colocation_set(xmlNode *set, int score, const char *coloc_id,
              xml_rsc != NULL; xml_rsc = crm_next_same_xml(xml_rsc)) {
 
             xmlNode *xml_rsc_with = NULL;
-            bool influence = true;
+            uint32_t influence = true;
```

Comment:
Set to `pcmk__coloc_has_influence` for clarity

Reply:
Whoops, oversight